server/routers/api.ts

```diff
@@ -24,18 +24,61 @@ interface ManifestAccessTicket {
 }
 const manifestAccessTickets: Map<string, ManifestAccessTicket> = new Map();
 
+function log(content: types.LogMessageSimple, context) {
+  let defaults: types.LogMessage = {
+    "event_version": "1",
+    "asctime": new Date().toISOString(),
+    "name": "shl-ltt-server",
+    "level": "INFO",
+    "deployment": env.DEPLOYMENT,
+    "system-type": "server",
+    "system-url": env.PUBLIC_URL,
+    "user": "",
+    "subject": "",
+    "tags": [],
+    "message": "",
+    "session-id": "",
+    "ip-address": context.request.ip,
+    "user-agent": context.request.userAgent,
```

Comment:
nice to see all this info captured! the fields names may change a bit, but that can be fixed later